components/brave_ads/browser/bundle_state_database.cc

```diff
@@ -342,24 +342,29 @@ bool BundleStateDatabase::CreateAdConversionsTable() {
   // new constraints to the schema
   const std::string sql = base::StringPrintf(
       "CREATE TABLE %s "
-          "(id INTEGER PRIMARY KEY, "
-          "creative_set_id LONGVARCHAR NOT NULL, "
+          "(creative_set_id LONGVARCHAR NOT NULL, "
           "type LONGVARCHAR NOT NULL, "
           "url_pattern LONGVARCHAR NOT NULL, "
-          "observation_window INTEGER NOT NULL)",
+          "observation_window INTEGER NOT NULL, "
+          "expiry_timestamp TIMESTAMP, "
+          "UNIQUE(creative_set_id, type, url_pattern, "
+              "observation_window), "
+          "PRIMARY KEY(creative_set_id, type, url_pattern, "
```

Comment:
should `expiry_timestamp` be part of the PK to allow for a new conversion with an updated expiry date to be saved in the case that a campaign date updated in flight?

Reply:
do we want duplicate entries with different timestamps and observation windows?